course/templates/course/index.html

```diff
@@ -12,85 +12,144 @@ <h1>{{ welcome_text|safe }}</h1>
 </div>
 {% endif %}
 
-<div class="row course-thumbnails">
-    {% site_advert as advert %}
-    {% if advert %}
+{% if my_instances %}
+<div class="frontpage-panel">
+    <h2 class="panel-heading">{% trans "My courses" %}</h2>
+    <div class="row course-thumbnails">
+        {% for instance in my_instances %}
+        {% with instance_url=instance|url %}
         <div class="col-sm-6 col-md-4 col-lg-3">
-            <div class="thumbnail advert-thumbnail">
-                <div class="image" aria-hidden="true">
-                    {% if advert.image %}
-                        <img src="{{ advert.image }}" alt="">
-                    {% else %}
-                        <h1 class="glyphicon glyphicon-comment"></h1>
-                    {% endif %}
-                </div>
+            <div class="thumbnail course-thumbnail">
+                <a href="{{ instance_url }}">
+                    <div class="image" aria-hidden="true">
+                        {% if instance.image %}
+                        <img src="{{ instance.image.url }}" alt="{{ instance.course.name|parse_localization }}" />
```

Comment:
I persoanlly prefer that block inside `if` is indented too. I need to look if django has any style guides for that...

Reply:
There were some mixed examples in the current source code. Also within the current index.html-file: 
With:
https://github.com/apluslms/a-plus/blob/master/course/templates/course/index.html#L21
Without:
https://github.com/apluslms/a-plus/blob/master/course/templates/course/index.html#L47

So my code was a bit mixed there as well. I couldn't find a source for the correct way for this in the django-documentation, but definitely we should have a common way of doing the indentation.

Reply:
As talked #elsewhere, current codebase has quite different styles in different files, thus we should slowly fix that. To be written in the style guid, html should be indented with a tab character and tags and elements both add to the indentation.

Reply:
These should be fixed. There might be still parts that use spaces instead of tabs (like talked #elsewhere) but at least they appear correct.

Reply:
> but at least they appear correct.

People usually have tab width set to 2 or 4 spaces, thus it becomes pretty clear when it's rendered with 8 spaces by git and github. Hence, if it looks nice in the editor and here, then it should be fine.